src/chart/line/LineView.ts

```diff
@@ -1299,7 +1299,10 @@ class LineView extends ChartView {
                 });
             }
             if (valueAnimation) {
-                labelInner(endLabel).setLabelText(value);
+                const inner = labelInner(endLabel);
```

Comment:
The existence check for `labelInner` seems unnecessary.
https://github.com/apache/echarts/blob/master/src/util/model.ts#L736C1-L741

Reply:
@Ovilia How about merging this quick fix into v5.4.3 release?